index.rst

```diff
@@ -42,20 +42,18 @@
 
```

Comment:
Patrick and I are adding a note at the beginning as a placeholder for an abstract. May be a good idea to add something similar here.

Reply:
OK. Will do.